src/blockchain/blockchain_behavior.cpp

```diff
@@ -58,6 +71,19 @@ const Settings &Behavior::GetDefaultSettings() const {
   return m_parameters.default_settings;
 }
 
+std::size_t Behavior::GetTransactionWeight(const CTransaction &tx) const {
```

Comment:
Is there any test for those functions? Cause I'd love to have some.

Reply:
I can add some.

Reply:
Added in https://github.com/dtr-org/unit-e/pull/1106/commits/8493b788fc1c9d89401f11a3d3e626fded965e86